src/jit/codegencommon.cpp

```diff
@@ -1269,18 +1269,14 @@ unsigned CodeGenInterface::InferStructOpSizeAlign(GenTree* op, unsigned* alignme
 bool CodeGen::genCreateAddrMode(GenTree*  addr,
                                 int       mode,
```

Comment:
Seems like `mode` is also not used. Can we get rid of it and the comment chunk it inspires?